packages/vanilla-components/package.json

```diff
@@ -29,6 +29,8 @@
     "apollo-client": "^2.3.5",
     "apollo-link-http": "^1.5.4",
     "clipboard": "^1.7.1",
+    "country-flags-svg": "^0.2.12",
+    "currency-flags": "git+https://github.com/transferwise/currency-flags.git",
```

Comment:
This seems like it will add significant bundle size bloat -- can you check how much our bundle size changes?

Reply:
Ah, the github one was a different attempt. `country-flags-svg` doesnt actually contain any svg's, it just maps links to the flags pulled from upload.wikimedia.org. That said, this implementation isn't 100% reliable due to not having access to an ISO country code from rewardbalances, so its likely to change.